js/src/sdk/base.toolset.ts

```diff
@@ -29,6 +25,14 @@ export class ComposioToolSet {
     localActions: IPythonActionDetails["data"] | undefined;
     customActionRegistry: ActionRegistry;
 
+    processors: {
+        pre?: (input: Record<string, any>) => Record<string, any>;
```

Comment:
Consider adding type safety for the processor input/output:

```typescript
type ProcessorInput = Record<string, unknown>;
type ProcessorOutput = Record<string, unknown>;

processors: {
    pre?: (input: ProcessorInput) => ProcessorOutput;
    post?: (input: ProcessorInput) => ProcessorOutput;
} = {
    pre: (input: ProcessorInput) => input,
    post: (input: ProcessorInput) => input
}
```

This makes the types more explicit and prevents using `any`.